S3/English/ASS_29.ass

```diff
@@ -0,0 +1,105 @@
+Style: Title,Arial,75,&H006CC2E9,&H000000FF,&H00000000,&H00000000,0,0,0,0,100,100,0,0,1,2,2,2,10,10,280,1
+
+[Events]
+Format: Layer, Start, End, Style, Name, MarginL, MarginR, MarginV, Effect, Text
+Dialogue: 0,0:01:02.51,0:01:05.56,Title,,0,0,0,, {\pos(970,920)} {\b1\an5\pos(970,880)\fs120\frz4.5\t(0,100,1.5\fs45)}Episode 29:  {\c&H7990E7&}Be aware of stray cats{\b0}
+Dialogue: 0,0:01:10.15,0:01:11.75,Default,,0,0,0,,Stop barking, move!
+Dialogue: 0,0:01:11.75,0:01:12.18,Default,,0,0,0,,Move!
+Dialogue: 0,0:01:17.00,0:01:17.80,Default,,0,0,0,,Is this...?
+Dialogue: 0,0:01:19.56,0:01:20.27,Default,,0,0,0,,Ira
+Dialogue: 0,0:01:20.38,0:01:21.02,Default,,0,0,0,,Look at this
+Dialogue: 0,0:01:21.65,0:01:23.20,Default,,0,0,0,,I found a cat downstairs
+Dialogue: 0,0:01:23.48,0:01:24.60,Default,,0,0,0,,It has two tails
+Dialogue: 0,0:01:24.85,0:01:25.62,Default,,0,0,0,,So unusual
+Dialogue: 0,0:01:25.93,0:01:26.84,Default,,0,0,0,,Isn't it...?
+Dialogue: 0,0:01:27.20,0:01:28.17,Default,,0,0,0,,I want to raise it
+Dialogue: 0,0:01:28.60,0:01:30.73,Default,,0,0,0,,But not sure if Lynn allows it
+Dialogue: 0,0:01:30.10,0:01:31.32,Default,,0,0,0,,{\pos(970,200)}What a great smell of alcohol
+Dialogue: 0,0:01:32.96,0:01:33.72,Default,,0,0,0,,First of all
+Dialogue: 0,0:01:33.72,0:01:35.36,Default,,0,0,0,,It's a nekomata, not a cat
+Dialogue: 0,0:01:35.62,0:01:36.00,Default,,0,0,0,,Second
+Dialogue: 0,0:01:36.40,0:01:37.54,Default,,0,0,0,,It's our next door in 1032
+Dialogue: 0,0:01:37.91,0:01:38.75,Default,,0,0,0,,You'd better put it back
+Dialogue: 0,0:01:38.85,0:01:39.76,Default,,0,0,0,,Before it wakes up
+Dialogue: 0,0:01:40.28,0:01:41.52,Default,,0,0,0,,In the corridor?
+Dialogue: 0,0:01:42.17,0:01:43.31,Default,,0,0,0,,Will it gets lost?
+Dialogue: 0,0:01:43.91,0:01:45.18,Default,,0,0,0,,And it never woke up
+Dialogue: 0,0:01:45.57,0:01:46.78,Default,,0,0,0,,Did it pass out?
+Dialogue: 0,0:01:48.12,0:01:49.14,Default,,0,0,0,,Who are you
+Dialogue: 0,0:01:56.11,0:01:57.62,Default,,0,0,0,,Why is it so aggressive
+Dialogue: 0,0:01:58.60,0:02:00.18,Default,,0,0,0,,And did it just speak?
+Dialogue: 0,0:02:00.70,0:02:02.00,Default,,0,0,0,,You learned something, Neil
+Dialogue: 0,0:02:02.60,0:02:04.15,Default,,0,0,0,,Nekomata is cat spirit in Japan
+Dialogue: 0,0:02:04.40,0:02:05.74,Default,,0,0,0,,Pretty normal that it speaks
+Dialogue: 0,0:02:08.60,0:02:09.70,Default,,0,0,0,,Don't run around
+Dialogue: 0,0:02:11.80,0:02:13.30,Default,,0,0,0,,That's my Lafite '82
+Dialogue: 0,0:02:13.33,0:02:14.44,Default,,0,0,0,,Damao's Vodka
+Dialogue: 0,0:02:14.44,0:02:16.62,Default,,0,0,0,,No! Lynn's red wine
+Dialogue: 0,0:02:19.60,0:02:21.13,Default,,0,0,0,,It's... drunk
+Dialogue: 0,0:02:21.27,0:02:21.64,Default,,0,0,0,,No
+Dialogue: 0,0:02:21.77,0:02:22.52,Default,,0,0,0,,It had drink so much wine
+Dialogue: 0,0:02:22.66,0:02:23.50,Default,,0,0,0,,Must take it to the hospital
+Dialogue: 0,0:02:31.70,0:02:32.62,Default,,0,0,0,,It becomes a woman?
```

Comment:
"It became a woman?" or "It turned into a woman?" (I think he says that after she transforms so it means it just happened but still placed in the past)